src/doc/bootstrap

```diff
@@ -89,8 +89,8 @@ fi
 OUTPUT_INDEX="$OUTPUT_DIR"/index.rst
 cat > "$OUTPUT_INDEX" <<EOF
 
-External Packages
-=================
+Features, Distributions, and External Packages
+==============================================
 
```

Comment:
How about just "Packages and Features"?

And we list them in the order

- standard packages
- optional packages
- distribution packages
- features
- experimental packages (perhaps after features)
- all packages (I don't know what to do with this item. Maybe removed later)